<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config</class>
 <widget class="QMainWindow" name="config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuracion del juego</string>
    </property>
    <widget class="QSpinBox" name="playTime">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiempo de partida (segundos): </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuracion de puerto serie</string>
    </property>
    <widget class="QComboBox" name="stopCombo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string>2</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bits de stop: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="dataCombo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudios:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Paridad:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Puerto:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="portCombo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="baudsCombo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bits de datos: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="parityCombo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>Par</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Impar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sin paridad</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
